extension-manifest-v3/src/background/serp.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { store } from 'hybrids';
+import {
+  tryWTMReportOnMessageHandler,
+  isDisableWTMReportMessage,
+} from '@ghostery/trackers-preview/background';
+import css from '@ghostery/trackers-preview/content_scripts/styles.css?raw';
+
+import Options from '/store/options.js';
+
+const SERP_URL_REGEXP = /google\.[a-z]+(\.[a-z]+)?\/search/;
```

Comment:
At other places, we are using patterns like that to avoid false-positives:

`^https:[/][/][^/]*[.]google[.]`

So, this more conservative regexp should still match:
```
/^https:[/][/][^/]*[.]google[.][a-z]+([.][a-z]+)?[/]search/
```